client/src/components/main/auth_modal/Register.js

```diff
@@ -39,22 +39,21 @@ export default props => {
       onSubmit={values => registerUser({ variables: values })}
     >
       <Form className="signup-form">
-        <label htmlFor="email">Email:</label>
```

Comment:
I really don't love you changing so many things on my components without having a discussion about it. If you don't like how I've done the form then we should talk about it as a group rather than just making the changes, what if both John and I preferred it the old way, or what if all three of us had a different idea of how to make this look best?

Reply:
Sorry about that, I should have discussed with you.